scripts/image-builder/build.sh

```diff
@@ -90,12 +92,12 @@ download_image() {
     local uuid=$1
     local logsonly=$2
 
-    sudo composer-cli compose logs ${uuid}
-    if [ -z "$logsonly" ] ; then
-        sudo composer-cli compose metadata ${uuid}
-        sudo composer-cli compose image ${uuid}
+    sudo composer-cli compose logs "${uuid}"
+    if [ -z "${logsonly}" ] ; then
+        sudo composer-cli compose metadata "${uuid}"
+        sudo composer-cli compose image "${uuid}"
     fi
-    sudo chown -R $(whoami). "${BUILDDIR}"
+    sudo chown -R "$(whoami)". "${BUILDDIR}"
```

Comment:
It's first time I see `.` in such place. @ggiguash is this a typo or secret functionality?

Reply:
@pmtk , it's a "secret" functionality :). It means that user's main group is used by the command automagically.